core/src/main/java/io/grpc/ClientCall.java

```diff
@@ -173,6 +173,25 @@ public void onReady() {}
   public abstract void sendMessage(ReqT message);
 
   /**
+   * Defers flushing the messages written using {@link #sendMessage(Object)} to the
+   * transport until the write buffer is full.  The amount of data buffered while corked is
+   * dependent on the transport implementation so users of this API should not make strong
+   * assumptions about how long data is buffered while corked.
+   *
+   * <p>Corking can provide throughput gains by allowing a transport to perform fewer writes to
+   * the underlying network.
+   */
+  @ExperimentalApi
+  public abstract void cork();
```

Comment:
Should these be noops?  Adding abstract methods to a public class will break people right?